templates/app/components/product_card_component.html.erb

```diff
@@ -40,15 +40,15 @@
       class="dark:text-black <%= text_flex_classes %>"
     >
       <header class="product-card_header <%= title_text_classes %>">
-        <h2>
+        <span>
           <%= link_to(
             truncate(product.name, length: 50),
             product_path(product),
             class: 'info',
             itemprop: "name",
             title: product.name
           ) %>
-        </h2>
```

Comment:
Given the goal here is to stop using H1-3 elements for content we don't want indexed, I have a question: Don't we want product names indexed?

Reply:
Yes, absolutely on product pages. 

On taxons we want to give preference to content created in the taxon (as on pages). Products are passed on via structured data.